zetaclient/tss/tss_keysign_manager.go

```diff
@@ -0,0 +1,49 @@
+package tss
+
+import (
+	"sync"
+
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+)
+
+// KeySignManager is a manager to keep track of concurrent keysigns performed by go-tss
```

Comment:
I can rename the struct, it is too generic for what it is actually doing. Having a separate structure for this does make it a bit cleaner because it requires a mutex and some logic. It will also allow us to expand on it later on if we want to keep track of other metrics like average keysign time etc.